fe/fe-core/src/main/java/com/starrocks/task/PushTask.java

```diff
@@ -91,7 +91,7 @@ public PushTask(TResourceInfo resourceInfo, long backendId, long dbId, long tabl
         this.transactionId = transactionId;
         this.tBrokerScanRange = null;
         this.tDescriptorTable = null;
-        this.useVectorized = false;
+        this.useVectorized = true;
```

Comment:
Unused variable

Reply:
PushTask is also used by SparkLoad, this is meaningful